app/viewmodels/helpers/ItemSmallIndependentProducerHelper.scala

```diff
@@ -76,11 +76,6 @@ class ItemSmallIndependentProducerHelper @Inject()(govukInput: GovukInput) {
           id = Some(s"${form(producerField).id}-${NotAIndependentSmallProducer.toString}"),
           value = Some(NotAIndependentSmallProducer.toString),
           content = Text(messages(s"itemSmallIndependentProducer.${NotAIndependentSmallProducer.toString}"))
```

Comment:
These should probably get refactored to be `NotApplicable` instead of `NotAIndependentSmallProducer` - just so that they make more sense now that the meaning of radio button has changed.